pandas/tests/groupby/test_groupby.py

```diff
@@ -2086,47 +2086,45 @@ def test_median_empty_bins(self):
         expected = df.groupby(bins).agg(lambda x: x.median())
         assert_frame_equal(result, expected)
```

Comment:
can you add a tests with the numeric ops on groupby / object and assert they raise

Reply:
Assuming you want this in a different change no problem. That said, do you consider the numeric ops to be `add`, `prod`, `min`, `max`, `mean`, `median`, `var`, `ohlc`, `cumprod`, `cumsum`, `cummin`, `cummax` and `rank`? The tests are one thing, but these don't all raise at the moment so would have to couple that with some refactoring of the `groupby` module

Reply:
yes let's do this in another change.